### DISCUSSION TITLE
Not showing images at all

### DISCUSSION THREAD

**Discussion by siddharth-upamanyu** [yifaneye/react-gallery-carousel#34] [Q&A] [answered]
Hi, I have been trying to consume this library but I am struggling with showing images. 
My image object is the same as yours. 
The component renders like this 
![Screenshot](https://user-images.githubusercontent.com/61269387/118400907-3d468400-b681-11eb-94fb-bbd0d9d9d53d.png)

This is my carousel component 
```
<Carousel
          images={images}
          isRTL
          isMaximized={false}
          hasThumbnails={false}
          style={{ height: 200 }}
          canAutoPlay={false}
          hasLeftButton={false}
          hasRightButton={false}
          hasSizeButton={false}
          hasIndexBoard={false}
          hasDotButtonsAtMax="bottom"
        />
```

**Comment by yifaneye** [comment]
Hi @siddharth-upamanyu !

I am sorry to hear that! 

I am here to help! 😉 

Would you like to check:
1. whether there is error or warnings in your browser console?
2. whether you imported this library correctly?
3. whether you are using TypeScript?

**Reply by siddharth-upamanyu** [reply]
Hey @yifaneye, Thanks for the prompt reply.
So, I am using typescript and to overcome the type error, I had to add "noImplicitAny" as false. 
What will be the correct way to use it in typescript if at all possible ?

**Reply by yifaneye** [reply]
Nice to hear that we found the error and the solution! The correct way is to write a type definition for this package. I will start writing one this week.

**Reply by siddharth-upamanyu** [reply]
@yifaneye Great! But that means I can't use it as of now? Or is there a way to still use it with typescript?
Because even though now I don't see the type error, I am still not seeing the images in the carousel.

**Reply by yifaneye** [reply]
I will try to make a carousel using TypeScript tonight! I will let you know!

**Reply by yifaneye** [reply]
This is what I am now getting on a TypeScript setup using create-react-app's TypeScript template. To solve it, I will add type declaration this week or the next.
![SS 2021-05-18 at 02 19 25](https://user-images.githubusercontent.com/49315663/118522552-8977ee00-b77f-11eb-9bc1-631bcbf61d1d.jpg)

**Reply by yifaneye** [reply]
Hi @siddharth-upamanyu , the TypeScript type definition for react-gallery-carousel is now available through DefinitelyTyped. Please see https://www.npmjs.com/package/@types/react-gallery-carousel

**Comment by yifaneye** [comment, marked as answer]
Hi @siddharth-upamanyu , the problem had to do with the lack of TypeScript support. Now, TypeScript type definitions for react-gallery-carousel is now available through DefinitelyTyped. Please see https://www.npmjs.com/package/@types/react-gallery-carousel
